src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -510,18 +516,24 @@ public void Visit(SensitivityAnalysis sensitivityAnalysis)
 
       public void Visit(ParameterIdentification parameterIdentification)
       {
+         _currentlyActivePI = parameterIdentification.Id;
+         _parameterIdentificationRunning = _activePIs.Contains(parameterIdentification.Id);
```

Comment:
Sorry i don't understand any of this code.
Why do you need these flags? 
`_parameterIdentificationRunning ` seems to only be set and never used?

Reply:
ok I see it's being used below. But you can start multiple PI at the same time. Why do you need to know that one is running?

Reply:
`_parameterIdentificationRunning` is used to set the menu view accordingly. You can check the `updateParameterIdentificationItems` method where the buttons are enabled and disabled based on its value.